### ISSUE TITLE
Doesnt work?

### ISSUE THREAD

**Issue by TherealONE12** [noto-rious/Nitro.Self-V3#20]
So basacly The Sniper looks like this 
![image](https://github.com/noto-rious/Nitro.Self-V3/assets/144170391/f75cf642-dca0-4e0f-b5ed-0246fd6fe93f)
and does NOT snipe Giveaways/Nitro coodes 
![Screenshot 2023-10-22 103837](https://github.com/noto-rious/Nitro.Self-V3/assets/144170391/9b9b6cf1-45fb-4842-add1-d3c0fc499885)
This is my Config file. Also, when i enable "Sniping on main " ,The tool doesnt starts. Can sw Help pls? I downloaded the newest version under Releases for Windows.

**Comment by TherealONE12** [comment]
Now i get more errors: 
![image](https://github.com/noto-rious/Nitro.Self-V3/assets/144170391/f82f0a4d-d4b8-4a42-805a-79dd66b1f113)
Pls help

**Comment by TherealONE12** [comment]
Anyone here?

**Comment by thekillercrum** [comment]
yup, same error

**Comment by TherealONE12** [comment]
> yup, same error
 Nice. sshould I try and fix it? 
Maybe im able to do it...

**Comment by TherealONE12** [comment]
> yup, same error

u know wich format the Nitro.self-3 thing has? (.py, .txt, ...)

**Comment by thekillercrum** [comment]
yes please try fix it also add me on discord as i wont check this often : thekillercrum

i did manage to get it to work if this is the same on i fixed

**Comment by TherealONE12** [comment]
> i did manage to get it to work if this is the same on i fixed
 What?
